Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -271,6 +271,18 @@ void TextDocument::insert_line(size_t line_index, NonnullOwnPtr<TextDocumentLine
     }
 }
 
+void TextDocument::swap_lines(size_t a, size_t b)
+{
+    auto tmp = m_lines[(int)a];
+    m_lines[(int)a] = m_lines[(int)b];
+    m_lines[(int)b] = tmp;
```

Comment:
What's up with the `(int)` casts here? The subscript operator of `m_lines` takes `size_t` already, no?